src/platform/qt/library/LibraryController.cpp

```diff
@@ -37,6 +37,13 @@ void AbstractGameList::updateEntry(const LibraryEntry& item) {
 void AbstractGameList::removeEntry(const QString& item) {
 	removeEntries({item});
 }
+void AbstractGameList::setShowFilenameInLibrary(bool newValue) {
```

Comment:
```suggestion
void AbstractGameList::setShowFilename(bool showFilename) {
```
Let's make it a little less verbose, and the parameter a bit more descriptive.